_tutorials/page8.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: tutorial
+title: Nested Endpoints
+name: Nested Endpoints
+previous_page_title: Chaining Requests
+next_page_title: Default Values
+---
+
+# Nested Endpoints
+
+In order to clean and reuse our code, one `endpoint` can have multiple `endpoints`. This way,
+we can aggregate requests that have the same route.
+
+Let's see how we could rewrite our specification file `scanapi.yaml` using nested endpoints:
+
+```yaml
+{% raw %}endpoints:
```

Comment:
Is it possible to highlight certain linenos? If yes, I think it would be cool to have it.

Reply:
I could not find a simple way to do it 😢 it would be much better...